lucille-examples/lucille-distributed-example/conf/main.conf

```diff
@@ -0,0 +1,86 @@
+# this ingest runs two connectors in sequence

```

Comment:
this original version of this config in the file-to-file-example was heavily commented for the purpose of that example. I don't think we should duplicate all the comments here. The comments in this version are likely to become stale.